### ISSUE TITLE
[Bug]: Error message appears in SMAPI after loading the game

### ISSUE THREAD

**Issue by Pup1L-1510** [Annosz/UIInfoSuite2#607]
### Mod Version

v2.3.4

### Multiplayer

Single Player

### Steps to Reproduce
1. Open Stardew Valley
2. Load
3. When in the game UI that reminding that the days remains of the upgrading pan disappeared
4. Open the SMAPI tab the red error message is constantly reporting
![image](https://github.com/user-attachments/assets/84f88a97-2b78-49d6-88d2-5f32e71bd52c)

**Comment by NotCaiden** [comment]
I have encountered the same problem, is there any solution？

**Comment by drewhoener** [comment]
Please use the debug build in #560, sorry. Haven't gotten to posting it yet since I'm in the middle of a rewrite

**Comment by Pup1L-1510** [comment]
> I have encountered the same problem, is there any solution？

yeah, this error will only appears when you upgrading your Pan, cuz the error message is about "ShowToolUpgradeStatus", my way to dealing this was just off the status icon showing when the pan is upgrading.

**Comment by Pup1L-1510** [comment]
> I have encountered the same problem, is there any solution？

yo nvm man, it's fixed alr in the latest version lmao :)))

**Comment by NotCaiden** [comment]
i see,Thanks<3
